components/technology/modules/ROOT/pages/differences.adoc

```diff
@@ -0,0 +1,103 @@
+= SKALE Chain Differences from Ethereum
+
+SKALE Chains operate like Ethereum, and there are a few other features and differences that make SKALE Chains even more powerful and suitable to run your dApp. 
+
+== Zero gas fee transactions
+
+Transactions on SKALE Chain run in without any gas fee. This means there is gas on SKALE Chains just like on Ethereum, but it's powered by what's called SKALE FUEL (sFUEL), which has no monetary value. In this way, executing transactions and state changes don't incur costs but require consuming gas in sFUEL, which is value-less. SKALE Chain gas provides a way to meter or limit transactions on the SKALE Chain to prevent malicious execution.
```

Comment:
Transactions on a SKALE Chain have zero gas fees. This model operates by using a gas token similar to Ethereum called sFUEL. SKALE Fuel (sFUEL) has no monetary value meaning that executing transactions and state changes do not incur a cost, but do consume the gas token (sFUEL). sFUEL provides a way to meter or limit transactions on a given SKALE Chain in order to prevent malicious execution.

Reply:
accepted